django_prices_openexchangerates/migrations/0001_initial.py

```diff
@@ -1,17 +1,12 @@
 # -*- coding: utf-8 -*-
 # Generated by Django 1.9.2 on 2016-03-17 09:36
-from __future__ import unicode_literals
```

Comment:
Are you sure that editing migrations is a good idea?

Reply:
I've tested this change locally with Django 2.0.1 on py35k and py36k and can confirm that `makemigrations` reports `No changes detected` after the removal of `__future__` import, something I'm consider farm more acceptable than editing `operations` content.